common/src/test/java/com/revenuecat/purchases/common/LogHandlerTest.kt

```diff
@@ -56,12 +56,17 @@ class LogHandlerTest {
 
     @Test
     fun debugWithDebugLogsDisabled() {
+        val enabled = debugLogsEnabled
+        debugLogsEnabled = false
```

Comment:
I needed to update this test introduced in #492 since the default is no longer necessarily `false`.